emmtypingtool/0.0.1/Dockerfile

```diff
@@ -65,4 +63,29 @@ RUN mkdir /db && cd /db && \
 
 ENV PATH="${PATH}:/samtools-0.1.19:/bowtie2-2.2.9:/EMBOSS-6.6.0/emboss:/emm-typing-tool"
 
-WORKDIR /data
\ No newline at end of file
+WORKDIR /data
+
+# Adding test layer
+FROM app as test
+
+ARG SRATOOLKIT_VER="3.0.2"
+ARG SRA_ACCESSION="ERR10762520"
+
+# Install SRA toolkit 
+RUN mkdir /test && cd /test && \
+wget https://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/${SRATOOLKIT_VER}/sratoolkit.${SRATOOLKIT_VER}-ubuntu64.tar.gz && \
+tar -vxzf sratoolkit.${SRATOOLKIT_VER}-ubuntu64.tar.gz && \
+rm sratoolkit.${SRATOOLKIT_VER}-ubuntu64.tar.gz
+
+ENV PATH="${PATH}:/test/sratoolkit.${SRATOOLKIT_VER}-ubuntu64/bin"
+
+# Download fastq files for test
+RUN mkdir /test/raw_reads && cd /test/raw_reads && \
+fasterq-dump --split-files ${SRA_ACCESSION} && \
+gzip ${SRA_ACCESSION}_*
+
+WORKDIR /data
+
+# Run emmtypingtool on test files
+RUN mkdir /data/test && \
+python /emm-typing-tool/emm_typing.py -m /db -i /test/raw_reads -o /data/test
```

Comment:
I like this test:
Downloads fastq files for ERR10762520 and runs them through /emm-typing-tool/emm_typing.py

Reply:
Thanks!